Cargo.lock

```diff
@@ -1927,7 +2368,40 @@ checksum = "6db9e6914ab8b1ae1c260a4ae7a49b6c5611b40328a735b21862567685e73255"
 dependencies = [
  "libc",
  "wasi 0.10.0+wasi-snapshot-preview1",
- "winapi",
+ "winapi 0.3.9",
+]
+
+[[package]]
+name = "tinyvec"
+version = "1.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "5b5220f05bb7de7f3f53c7c065e1199b3172696fe2db9f9c4d8ad9b4ee74c342"
+dependencies = [
+ "tinyvec_macros",
+]
+
+[[package]]
+name = "tinyvec_macros"
+version = "0.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "cda74da7e1a664f795bb1f8a87ec406fb89a02522cf6e50620d016add6dbbf5c"
+
+[[package]]
+name = "tokio"
+version = "0.2.25"
```

Comment:
Ugh well except something in the depchain is dragging in ancient tokio...

Reply:
OK fixed by bumping to `xmlrpc = "0.15.0"`.  But, this does in turn drag `reqwest/hyper` into the stack.  Which...well, we'd need a conversation about keeping around libcurl then.

Reply:
Since we just need it for a small subset, I wonder how hard it'd be to just re-implement the bits we need on top of libcurl.